packages/core/src/components/popover/popover.tsx

```diff
@@ -276,10 +279,21 @@ export class Popover extends AbstractPureComponent<IPopoverProps, IPopoverState>
         }
     }
 
+    private updatePopperObserver() {
+        if (this.popperObserver instanceof ResizeObserver) {
+            this.popperObserver.disconnect();
+        }
+
+        this.popperObserver = new ResizeObserver(() => this.popperScheduleUpdate && this.popperScheduleUpdate());
```

Comment:
seems to me that calling `disconnect()` should be enough, no need to recreate: https://wicg.github.io/ResizeObserver/#dom-resizeobserver-disconnect

> 1) Clear the observationTargets list.
>
> 2) Clear the activeTargets list.

please revert this so it's only created once and simply invoke `disconnect` before observing the new one.